docs/usage/oauth.md

```diff
@@ -6,7 +6,16 @@ To do that, you can follow the steps below.
 
 ## Add a route to start OAuth
 
-The route for starting the OAuth process (in this case `/login`) will use the library's `beginAuth` method.  The `beginAuth` method takes in the request and response objects (from the `http` module), along with the target shop _(string)_, redirect route _(string)_, and whether or not you are requesting [online access](https://shopify.dev/concepts/about-apis/authentication#api-access-modes) _(boolean)_. The online access boolean defaults to `true`.The method will return a URI that will be used for redirecting the user to the Shopify Authentication screen.
+The route for starting the OAuth process (in this case `/login`) will use the library's `beginAuth` method.  The `beginAuth` method takes in the request and response objects (from the `http` module), along with the target shop _(string)_, redirect route _(string)_, and whether or not you are requesting [online access](https://shopify.dev/concepts/about-apis/authentication#api-access-modes) _(boolean)_. The online access boolean defaults to `true`. The method will return a URI that will be used for redirecting the user to the Shopify Authentication screen.
```

Comment:
I think we may not even want to state all of this since we're going into details on each specific parameter in the table. We could just mention the most important points (like the `/login` part or the link to the online token docs) and go straight into the table.

Reply:
ohh okay yeah that works! I just copied the formatting of the PHP library docs (which is redundant in the same way)